charts/gafaelfawr/templates/ingress-token.yaml

```diff
@@ -6,7 +6,7 @@ metadata:
     nginx.ingress.kubernetes.io/auth-request-redirect: $request_uri
     nginx.ingress.kubernetes.io/auth-response-headers: X-Auth-Request-Token
     nginx.ingress.kubernetes.io/auth-signin: https://{{ .Values.host }}/login
-    nginx.ingress.kubernetes.io/auth-url: https://{{ .Values.host }}/auth?scope={{ .Values.user_scope }}
+    nginx.ingress.kubernetes.io/auth-url: http://gafaelfawr-service.{{ .Release.Namespace }}.svc.cluster.local:8080/auth?scope={{ .Values.user_scope }}
```

Comment:
Unforutnately, we can't use `svc.cluster.local` because it doesn't work on the base and summit clusters. I think you can just omit those parts.

Reply:
Hrm I think I tried that and it didn't work on minikube.  I'll test that out tomorrow.

Reply:
Maybe just pull out the whole line into a helm values thing?